docs/user/gcp/install_upi.md

```diff
@@ -24,6 +25,11 @@ Use a [staged install](../overview.md#multiple-invocations) to enable desired cu
 ### Create an install config
 Create an install configuration as for [the usual approach](install.md#create-configuration).
 
+If you are installing into a [Shared VPC][sharedvpc],
+the installer will not be able to find the base domain zone in the host project.
+You must either create the install-config another way, or select another zone in the service project and then update the `install-config.yaml` manually.
```

Comment:
I think we should request the user to create install-config.yaml manually using the docs references/examples
and request that the publish be set to Internal.

we can add a NOTE that this is a restriction today for XPN.

Reply:
Resolved in 657a91d789da80eb9e2e96efc4ded29f490d64a9